### PR TITLE
ingest: documentation for the drop processor

### PR DESCRIPTION
**Pull request by jakelandis**
part of #33188

------
image

### CODE REVIEW

File: docs/reference/ingest/ingest-node.asciidoc [hunk 1]
```diff
@@ -1242,6 +1242,21 @@ Reference key modifier example
 * error = REFUSED
 |====== 
 
+[[drop-processor]]
+=== Drop Processor
+Drops the document without raising any an errors. This is useful to prevent the document from
```
Comment (original-brownbear) [inline docs/reference/ingest/ingest-node.asciidoc]: Typo `s/an//`

**Review by original-brownbear — CHANGES_REQUESTED** [review]
One typo :)

File: docs/reference/ingest/ingest-node.asciidoc (hunk 1 above)
Comment (jakelandis) [reply docs/reference/ingest/ingest-node.asciidoc]: @original-brownbear - thanks! fixed.

**Review by original-brownbear — APPROVED** [review]
@jakelandis thanks LGTM

**Review by rjernst — APPROVED** [review]
LGTM too, just one ask.

**Comment by jakelandis** [comment]
6.5 backport: https://github.com/elastic/elasticsearch/commit/aed947f3f1edbae1021a503a12406fa4127febf6

**Pull request merged**